packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -354,15 +354,20 @@ export class RestAPIClient {
 			params
 		);
 
+		// This handles API errors such as `unauthorized`.
 		if ( response.hasOwnProperty( 'error' ) ) {
 			throw new Error(
 				`${ ( response as ErrorResponse ).error }: ${ ( response as ErrorResponse ).message }`
 			);
 		}
 
-		if ( response.errors.length === 0 ) {
-			console.log( response );
-			throw new Error( `Failed to create invite: ${ response.errors }` );
+		// This handles "errors" relating to the invite itself and can be an array.
+		// For instance, if a user tries to invite itself, or invite an already added user.
+		if ( response.errors.length !== 0 ) {
```

Comment:
nitpicky review comment alert :D

I kinda prefer `> 0` to `!== 0`, but it doesn't make a practical difference. Or, you can even just do `if ( response.errors.length )` for a quick truthy check

Reply:
True, I like that.